testing/log4cxx/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer: Russ Webber <russ@rw.id.au>
+pkgname=log4cxx
+pkgver=0.10.0
+pkgrel=0
+pkgdesc="A C++ port of the Log4j logging framework."
+url="http://logging.apache.org/log4cxx"
+license="Apache-2.0"
+options="!check"  # testsuite in 0.10.0 does not work on alpine
+subpackages="$pkgname-dev"
+depends=""
+arch="all"
+makedepends="autoconf automake libtool zip libxml2-dev apr-util-dev"
+source="http://archive.apache.org/dist/logging/$pkgname/$pkgver/apache-$pkgname-$pkgver.tar.gz
+        log4cxx-0.10.0-missing_includes.patch
+        log4cxx-0.10.0-narrowing-fixes-from-upstream.patch"
+builddir="${srcdir}/apache-$pkgname-$pkgver"
+
+prepare() {
+  update_config_guess
+  default_prepare
+}
+
+build() {
+  # if configure is autogenerated make install seems to fail #./autogen.sh
+  # does not configure if specifying --build=$CBUILD \ --host=$CHOST \
+  ./configure \
+    --prefix=/usr \
+    --sysconfdir=/etc \
+    --mandir=/usr/share/man \
+    --localstatedir=/var
+
+  make
+}
+
+check() {
+  make check
+}
+
+package() {
+  make DESTDIR="${pkgdir}" install
+}
+
+sha512sums="1c34d80983db5648bc4582ddcf6b4fdefdc6594c2769f95235f5441cd6d03cf279cc8f365e9a687085b113f79ebac9d7d33a54b6aa3b3b808c0e1a56a15ffa37  apache-log4cxx-0.10.0.tar.gz
```

Comment:
```suggestion
sha512sums="1c34d80983db5648bc4582ddcf6b4fdefdc6594c2769f95235f5441cd6d03cf279cc8f365e9a687085b113f79ebac9d7d33a54b6aa3b3b808c0e1a56a15ffa37  apache-log4cxx-0.10.0.tar.gz
4099b3ae2a927ccec2be2eee3a4e7a1dd4d979c7cb09f8a72b99059cb87a9e561249
71ec7daffbe13f925e42479d896cfeffe58521a2e66e04f565d3e5540d7b  log4cxx-GPGKEYS
```

Reply:
Does the github link need to be added?

Reply:
Nope. I put it there as a useful reference.

Reply:
Probably good thing as it isn't merged yet.

Reply:
Please try out the GPG support and leave your thoughts. It likely won't be merged before 3.10 but it has made things easier and safer when upgrading packages.